woocommerce/woocommerce-gutenberg-products-block
# All Reviews block

Fixes #847.

This PR:
* Creates an _All Reviews_ block.
* Makes the three review blocks share the same frontend script.
* Moves some editor logic from each block to shared files.

### Screenshots

![image](https://user-images.githubusercontent.com/3616980/63528544-716c2f80-c503-11e9-833f-b21f97f230e2.png)

### How to test the changes in this Pull Request:

1. Create a post and add an _All Reviews_ block.
2. Try modifying attributes and verify they are honored in the editor and the frontend.
3. Since this PR moves some code from other reviews blocks to shared files, it would be good to test _Reviews by Product_ and _Reviews by Category_ blocks to.

### Changelog

> A new block named 'All Reviews' was added in order to display a list of reviews from all products and categories of your store.
